evalml/tests/automl_tests/test_automl.py

```diff
@@ -142,7 +142,7 @@ def test_pipeline_limits(mock_fit_binary, mock_score_binary,
     automl.search(X, y)
     out = caplog.text
     assert "Will stop searching for new pipelines after 1 seconds" in out
-    assert len(automl.results['pipeline_results']) >= 1
+    assert len(automl.results['pipeline_results']) >= 5
```

Comment:
Just curious, is this >= 5 kind of arbitrary because of time = 1 sec?

Reply:
Yep, was arbitrary. I'll change it back